### DISCUSSION TITLE
In mirrored cluster mode, how to determine which node is the master node?

### DISCUSSION THREAD

**Discussion by jiuang** [rabbitmq/rabbitmq-server#5544] [Other]
I deployed the mirror-cluster mode for rabbitmq server. (3 nodes)
Now, I have a question. I don't know which node is the master node?

I attempted to find the master from "http://ip:15672/", and I also observed the information from the command 'rabbitmqctl cluster_status'.
However, I still couldn't find the master node.

Please give me some help.
Thank you so much.

**Comment by michaelklishin** [comment]
Consider using Discussions for questions and not issues.

All nodes in a RabbitMQ cluster [are equal peers](https://www.rabbitmq.com/clustering.html#cluster-membership). Replicated queue types display their set of replicas and the node that hosts the leader replica as part of their metrics, in particular in the management UI.

Classic mirrored queues are a deprecated feature that will be removed in RabbitMQ 4.0. Use quorum queues and/or streams. Quorum queues in 3.10 beat CMQs in every category. You can learn more on the [RabbitMQ blog](https://blog.rabbitmq.com): [one](https://blog.rabbitmq.com/posts/2022/05/rabbitmq-3.10-release-overview/), [two](https://blog.rabbitmq.com/posts/2022/05/rabbitmq-3.10-performance-improvements/).

**Reply by jiuang** [reply]
Copy that! Thank you very much!
